.buildkite/scripts/steps/cloud/purge_projects.ts

```diff
@@ -9,30 +9,37 @@
 
 import { execSync } from 'child_process';
 import axios from 'axios';
+import type { KibanaSolution } from '@kbn/projects-solutions-groups';
```

Comment:
This is a separate TS project local to `.buildkite`, the import isn't going to resolve